tests/containers/helm_k3s.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart
```

Comment:
It would be nice to have a better description, explaining the basic idea and if you want, a bullet list of the sub-tests, like in
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/buildah.pm#L6